google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneCloudRepositoryDialog.java

```diff
@@ -235,6 +235,7 @@ protected void textChanged(DocumentEvent event) {
         repositorySelector.setCloudProject(projectSelector.getText());
         repositorySelector.setUser(projectSelector.getSelectedUser());
         repositorySelector.setText("");
+        repositorySelector.loadRepositories(null /*onComplete*/);
```

Comment:
not sure. I'm on the fence about if I prefer being explicit about whether the callback is passed in or not. So I'll leave it unless you feel strongly.